src/types/quizzes.d.ts

```diff
@@ -16,38 +36,89 @@ declare class Quizzes {
   getQuizNextQuestion(
     id: string,
     parameters?: QuizzesParameters,
-    userParameters?: UserParameters,
     networkParameters?: NetworkParameters
   ): Promise<NextQuestionResponse>;
 
   getQuizResults(
     id: string,
-    parameters?: QuizzesParameters,
-    userParameters?: UserParameters,
+    parameters?: QuizzesResultsParameters,
     networkParameters?: NetworkParameters
   ): Promise<QuizResultsResponse>;
 }
 
 /* quizzes results returned from server */
 export interface NextQuestionResponse extends Record<string, any> {
-  next_question: Partial<NextQuestion>;
+  next_question: Question;
   is_last_question?: boolean;
-  version_id?: string;
+  quiz_version_id?: string;
+  quiz_id?: string;
+  quiz_session_id?: string;
```

Comment:
Does this mean they are optional? I believe they should be returned with every response right? 🤔